test/testdata/rbs/signatures_attrs.rb

```diff
@@ -45,7 +45,7 @@ def initialize
 T.reveal_type(x.foo) # error: Revealed type: `Integer`
 T.reveal_type(x.bar) # error: Revealed type: `T.nilable(Integer)`
 T.reveal_type(x.baz) # error: Revealed type: `T.nilable(Integer)`
-x.qux = "" # error: Assigning a value to `qux` that does not match expected type `T.nilable(Integer)`
+x.qux = "" # error: Assigning a value to `:qux` that does not match expected type `T.nilable(Integer)`
```

Comment:
This error message change is a little odd, do you know why it started including the `:` from the symbol after this change?

Reply:
I think the difference is that in rewriter we use the ident name loc while in pre-desugar we use the symbol loc which includes. 

I can actually change the offset to point to the name without the `:` prefix (see last commit) 👍